tests/index.test.js

```diff
@@ -52,4 +52,11 @@ describe('jest-fail-on-console', () => {
 
     expect(stderr).toEqual(expect.stringContaining(passString('assert-success')))
   })
+
+  it('does not print a stacktrace when disableStacktrace is true', async () => {
+    const { stderr } = await runFixture('option-disable-stacktrace')
+
+    expect(stderr).toEqual(expect.stringContaining('No additional stacktrace will be shown.'))
+    expect(stderr.length).toBeLessThan(1100); // Short output is the main point of this test
```

Comment:
I think the real current output of this test is 1074, and the other tests are more like 1400. This might break when we don't want it to, so maybe testing length here isn't a good idea.

Reply:
Could we check the number of lines of stderr? Maybe counting the `\n`?

Reply:
That's a good idea! I'll give that a try.